src/nopemd.h.in

```diff
@@ -290,13 +297,22 @@ NMDAPI int nmd_get_info(struct nmd_ctx *s, struct nmd_info *info);
  *
  * The function is blocking, it will make sure any asynchronous operation
  * previously requested (start, seek, stop) is honored before returning.
+ *
+ * Return:
+ *   - NMD_RET_NEWFRAME  a new frame is returned, frame pointer is not NULL
+ *   - NMD_RET_UNCHANGED the returned frame would be the same as the previous
+ *                       one, frame pointer is always NULL
+ *   - NMD_ERR_EOF       reached end of file (or requested end time), frame
+ *                       pointer is always NULL
+ *   - NMD_ERR_GENERIC   unable to return a frame because of an error, frame
+ *                       pointer is always NULL
```

Comment:
I'd rather have these comments in the enum declaration and use `Return: NMD_RET_* or NMD_ERR_*` (might make sense to have `NMD_RET_ERR_*` so that you could refer to them as `NMD_RET_*`). That way you'll avoid the duplication below.

Reply:
Ok. I don't mind but IHMO when you look at the doc or you idea gives you the signature + doc it adds a level of indirection.